handlers/charts.go

```diff
@@ -146,7 +146,11 @@ func SlotViz(w http.ResponseWriter, r *http.Request) {
 		Selector: "checklist",
 		Epochs:   services.LatestSlotVizMetrics(),
 	}
-	data.Data = slotVizData
+	data.Data = struct {
+		SlotVizData types.SlotVizPageData
+	}{
+		SlotVizData: slotVizData,
+	}
```

Comment:
**Minor:** Consider adding a comment why this is done the way it is done.

Reply:
comment added